content/source/docs/enterprise-beta/saml/configuration.html.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: enterprise2
+page_title: "SAML Configuration - Terraform Enterprise Beta"
+sidebar_current: "docs-enterprise2-saml-configuration"
+---
+
+# Configuration
+
+Go to `https://[your-terraform-domain]/admin/settings/saml/edit` and you’ll need to set the following values
```

Comment:
This url throws 404. The actual url is: 
https://[your-terraform-domain]/admin/settings/saml

Reply:
Thanks Ignas! You're right, I was going off an older pre release version.